task_geo/data_sources/__init__.py

```diff
@@ -1,13 +1,15 @@
 from task_geo.data_sources.covid.cds import cds
 from task_geo.data_sources.covid.nyt import nyt
 from task_geo.data_sources.demographics.us_census import us_census
+from task_geo.data_sources.hdx import hdx_acap_api
 from task_geo.data_sources.noaa import noaa_api
 
 AVAILABLE_DATA_SOURCES = {
     'noaa_api': noaa_api,
     'cds': cds,
     'us_census': us_census,
-    'nyt': nyt
+    'nyt': nyt,
+    'hdx': hdx_acap_api
```

Comment:
The name of the data source should be `hdx_acap`, not `hdx_acap`, the display name should be also the same as the datasource name, this line should be:

```python
AVAILABLE_DATA_SOURCES = {
    ...
    'hdx_acap': hdx_acap,
}
```
This means the module name, and the data source name should be changed too.